docs/execution-providers/ROCm-ExecutionProvider.md

```diff
@@ -0,0 +1,71 @@
+---
+title: ROCm (AMD)
+description: Instructions to execute ONNX Runtime with the AMD ROCm execution provider
+parent: Execution Providers
+nav_order: 11
+redirect_from: /docs/reference/execution-providers/ROCm-ExecutionProvider
+---
+
+# ROCm Execution Provider
+{: .no_toc }
+
+The ROCm Execution Provider enables hardware accelerated computation on AMD ROCm-enabled GPUs. 
+
+## Contents
+{: .no_toc }
+
+* TOC placeholder
+{:toc}
+
+## Install
+
+Pre-built binaries of ONNX Runtime with ROCm EP are published for most language bindings. Please reference [Install ORT](../install).
+
+## Requirements
+
+
+|ONNX Runtime|ROCm|
+|---|---|
+|main|5.2.3|
+|1.12|5.2.3|
+|1.12|5.2|
+
+
+## Build
+For build instructions, please see the [BUILD page](../build/eps.md#amd-rocm). 
+
+## Usage
+
+### C/C++
+
+```c++
+Ort::Env env = Ort::Env{ORT_LOGGING_LEVEL_ERROR, "Default"};
+Ort::SessionOptions sf;
+int device_id = 0;
+Ort::ThrowOnError(OrtSessionOptionsAppendExecutionProvider_ROCm(sf, device_id));
```

Comment:
It is weird that session options is abbreviated as `sf`, can we rename it to `so` or better
```cpp
Ort::ThrowOnError(OrtSessionOptionsAppendExecutionProvider_ROCm(Ort::SessionOptions{}, device_id));
```
since we are not passing any special optoin here.